e2e/BKTClient.spec.ts

```diff
@@ -60,4 +64,90 @@ suite('e2e/BKTClientTest', () => {
       reason: 'RULE',
     })
   })
+
+  suite('forceUpdate', () => {
+    test('evaluatedAt is too old', async () => {
+      const client = getBKTClient()
+
+      assert(client != null)
+
+      const evaluationStorage = getDefaultComponent(
+        client,
+      ).dataModule.evaluationStorage() as EvaluationStorageImpl
+
+      const current = evaluationStorage.storage.get()
+
+      assert(current != null)
+
+      // update evaluations manually, and check if evaluation1 is deleted after fetchEvaluations
+      evaluationStorage.storage.set({
+        ...current,
+        evaluations: { [evaluation1.featureId]: evaluation1 },
+        evaluatedAt: '1',
```

Comment:
It seems like just changing `evaluatedAt` to a super-old value does not make force-update.

this `/get_evaluations` request

```json
{
  "user":{"id":"bucketeer-js-user-id-1","attributes":{}},
  "userEvaluationsId":"3227641913513702639",
  "userEvaluationCondition":{"evaluatedAt":"1","userAttributesUpdated":false},
  "tag":"javascript",
  "sourceId":7,
  "sdkVersion":"2.0.2"
}
```

results in the response below

```json
{
    "state": "FULL",
    "evaluations": {
        "id": "no_evaluations",
        "evaluations": [],
        "createdAt": "1689779952",
        "archivedFeatureIds": [],
        "forceUpdate": false
    },
    "userEvaluationsId": "3227641913513702639"
}
```

Reply:
The condition is when the `userEvaluationsId` is different and the `evaluatedAt` is older than 31 days.
So we need to use a random string in the `userEvaluationsId` to make it work.

Sorry for my mistake.

Reply:
Thanks, fixed: [`4ac8be3` (#91)](https://github.com/bucketeer-io/javascript-client-sdk/pull/91/commits/4ac8be3adce5480619b0cd12a015d76ea0183807)

Reply:
Hi @yshrsmz  , I updated those changes to iOS. 
Is there a way to e2e test the `upsert evaluation` cases? I was looking at your PR. But I did not find it. 

cc: @cre8ivejp

Reply:
@duyhungtnn It's impossible to configure `archivedFeatureIds` and other stuff for e2e testing. So I just wrote forceUpdate cases.

`upsert` part is tested through unit tests, so it should be fine.

Reply:
thank you for confirming.